benchmarks/python/benchmark_inference.py

```diff
@@ -772,8 +835,9 @@ def parse_args() -> argparse.Namespace:
         default=None,
         help="Cache option: no caching, same input, constant values, symbolic values. See `cache` argument of `thunder.jit` for more details.",
     )
-    parser.add_argument("--enable-thunder-cudagraph", action="store_true", help="Pass CUDAGraphTransform to Thunder")
+    parser.add_argument("--enable-cudagraph", action="store_true", help="Pass CUDAGraphTransform to Thunder, or use reduce-overhead for torch.compile")
     parser.add_argument("--attn-implementation", type=str, default=None, help="Attention implementation")
+    parser.add_argument("--use-hardcoded-model", action="store_true", help="Use the hardcoded Llama4 config, rather than pulling from huggingfaces")
```

Comment:
Is it reasonable to have a single option to control the hardcoded config, and hardcoded model, in the future? I want to give this flag a name that will make sense when we add the hardcoded model. But if we need separate flags for these, we should name this flag something else.

Reply:
I don't have a strong opinion on this. Options are easy to change in benchmark scripts, so I'll let others with a stronger opinion to jump on this fight. 😜